sdk/search/Azure.Search.Documents/tests/Samples/FieldBuilder/AnalyzerAttribute.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Search.Documents.Indexes.Models;
+
+namespace Azure.Search.Documents.Tests.Samples
+{
+    /// <summary>
+    /// Indicates that the <see cref="SearchField"/> generated by <see cref="FieldBuilder"/> for
+    /// the target property should have its <see cref="SearchField.AnalyzerName"/> property set to the
+    /// specified analyzer.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property)]
```

Comment:
1. This was a port from track 1, and they didn't.
2. I could, but at this point I'd rather just have this be a straight port, since it's only to make migrating easier. The final `FieldBuilder` - really, the `ObjectSerializer` and friends - could, but is something we'd have to discuss further.